cli/cli/gateway.py

```diff
@@ -150,6 +159,61 @@ def get_routes(self) -> List[Route]:
 
         return routes
 
+    def get_consumers(self) -> List[Consumer]:
```

Comment:
[List is apparently deprecated](https://docs.python.org/3/library/typing.html#typing.List) I'm guilty of this in the api framework ^^

```suggestion
    def get_consumers(self) -> list[Consumer]:
```